validation/edge-validator/error-report.1.sample.fail.json

```diff
@@ -0,0 +1,26 @@
+{
+    "results": {
+        "namespace.doctype.1": {
+            "error_count": -1,
```

Comment:
Let's split the various failure cases into separate example pings (negative error count, >100 error rate, missing errors) to make sure any of the three cases cause the expected failure.